basement_bot/plugins/mock.py

```diff
@@ -22,19 +26,23 @@ def mock_string(string):
 
 @commands.command(name="sb")
 async def mock(ctx):
-    user_to_mock = ctx.message.mentions or None
+    user_to_mock = ctx.message.mentions[0] if ctx.message.mentions else None
```

Comment:
Will you check if they did multiple or just ignore all the other mentions?

Reply:
This is a neat idea to consider, but probably best to just ignore the others